src/app/query/stacks/balance/stacks-ft-balances.hooks.ts

```diff
@@ -114,7 +117,7 @@ export function useStacksFungibleTokenAssetBalance(contractId: string) {
       toast.error('Unable to find balance by contract id');
       navigate('..');
     }
-    return balance as StacksFungibleTokenAssetBalance;
+    return balance ?? createStacksFtCryptoAssetBalanceTypeWrapper(new BigNumber(0), contractId);
```

Comment:
Not sure if this is nec to change now with the route fix, but it was hitting this later as undefined in the form.